common/persistence/dataInterfaces.go

```diff
@@ -101,6 +101,7 @@ const (
 	TransferTaskTypeSignalExecution
 	TransferTaskTypeRecordWorkflowStarted
 	TransferTaskTypeResetWorkflow
+	TransferTaskTypeUpsertWorkflowSearchAttributes
```

Comment:
is UpsertSearchAttributes better than UpsertWorkflowSearchAttributes?

Reply:
I thought about it and consider UpsertWorkflowSearchAttributes more accurate, since we don't know whether we will have other type of search attributes like InsightSearchAttributes, DomainSearchAttributes and OperationSearchAtttibutes.